test/pytest/test_gRPC_inference_api.py

```diff
@@ -7,7 +7,7 @@
 import test_utils
 
 
-inference_data_json = "/../postman/inference_data.json"
+inference_data_json = "../postman/inference_data.json"
```

Comment:
Here also needs to be changed.

Reply:
it works well in joins, the output out put may seems weird, but it still work over Windows.
It would look like `Path\To\dir/../postman/infer.json` which works somehow with the simple `open` function of python